src/VisualStudio/Xaml/Impl/Implementation/LanguageServer/Handler/Completion/CompletionHandler.cs

```diff
@@ -42,6 +43,15 @@ public CompletionHandler()
 
         public override async Task<CompletionList?> HandleRequestAsync(CompletionParams request, RequestContext context, CancellationToken cancellationToken)
         {
+            if (request.Context is VSCompletionContext completionContext)
+            {
+                if (completionContext.InvokeKind == VSCompletionInvokeKind.Deletion)
+                {
```

Comment:
You can combine these with &&

Reply:
yeah good point.

---
In reply to: [618730489](https://github.com/dotnet/roslyn/pull/52848#discussion_r618730489) [](ancestors = 618730489)